marray/__init__.py

```diff
@@ -358,16 +358,15 @@ def get_linalg_fun(name):
         def linalg_fun(x1, x2, /, **kwargs):
             x1 = asarray(x1)
             x2 = asarray(x2)
-            data1 = xp.asarray(x1.data, copy=True)
-            data2 = xp.asarray(x2.data, copy=True)
-            data1[x1.mask] = 0
-            data2[x2.mask] = 0
+            zero = xp.asarray(0)
```

Comment:
Looks like this is unused, no?

Reply:
Oops, yes. I added it while writing and realized that I would need different zero types because   `where` of the Array API Standard doesn't support Python scalars. I'll remove it.

Reply:
I wonder why unused variable wasn't caught by linting. We need to enable that rule.